src/egon/data/datasets/power_plants/__init__.py

```diff
@@ -976,6 +977,156 @@ def fill_missing_bus_and_geom(gens, carrier):
 
         return gens
 
+    # Write nuclear power plants in supply.egon_power_plants
+    nuclear = pd.read_csv(
+        cfg["sources"]["mastr_nuclear"],
+        usecols=[
+            "EinheitMastrNummer",
+            "Energietraeger",
+            "Nettonennleistung",
+            "Laengengrad",
+            "Breitengrad",
+            "Gemeinde",
+            "Inbetriebnahmedatum",
+        ],
+    )
+    # drop generators installed after 2019
+    nuclear["Inbetriebnahmedatum"] = pd.to_datetime(
+        nuclear["Inbetriebnahmedatum"]
+    )
+    nuclear = nuclear[nuclear["Inbetriebnahmedatum"] < "2020-01-01"]
```

Comment:
It would be better to use the config parameter for this, as it is e.g. done here: https://github.com/openego/powerd-data/blob/60a4ced92d6fcd9635af6dcc95be75e0d9c16ef9/src/egon/data/datasets/chp/__init__.py#L344-L347

Reply:
Agree. Solved in: f18af17104cc87a18c31858c1d0c3bde8a35668d